frontend/src/pages/appTaskManagement/components/AppTable/TaskHistoryModal/TaskHistoryContainer.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { selectAppTaskHistoryData } from '../../../slice';
+
+function TaskHistoryContainer() {
+    const [visibleCodeIndex, setVisibleCodeIndex] = useState(null);
+    const TaskHistoryData = useSelector(selectAppTaskHistoryData);
+
+    console.log(TaskHistoryData);
+
+    const handleToggleCode = (index) => {
+        setVisibleCodeIndex(visibleCodeIndex === index ? null : index);
+    };
+
+    return (
+        <div className="overflow-y-auto">
+           {console.log('TaskHistoryData',TaskHistoryData)}
```

Comment:
@prem056627  remove console log

Reply:
@prem056627 Can you please remove this ?